src/components/Modals/CourseMenu.vue

```diff
@@ -42,8 +42,8 @@
           <span class="courseMenu-text">Edit Credits</span>
         </div>
         <img v-if="!isLeft" class="courseMenu-arrow" src="@/assets/images/sidearrow.svg" />
-         <div v-if="displayEditCourseCredits" class="courseMenu-content courseMenu-editCredits courseMenu-centerCredits"
-         :class="{'courseMenu-editCredits--left': isLeft }">
+          <div v-if="displayEditCourseCredits" class="courseMenu-content courseMenu-editCredits courseMenu-centerCredits"
```

Comment:
Ok this is super nitpicky - but was this improperly indented before or did you accidently change it?

Reply:
i think every indentation change i made was because my extension "indent rainbow" shows me it's incorrect - this is how it looked before i changed it.
<img width="926" alt="Screen Shot 2020-10-18 at 9 24 53 PM" src="https://user-images.githubusercontent.com/55263191/96391905-5db06d80-1188-11eb-98f0-674d96fa9628.png">